### ISSUE TITLE
Modernizr alternative

### ISSUE THREAD

**Issue by ajmueller** [isitedesign/Phoenix#53]
Browsernizr could be a possible replacement for a local version of Modernizr.  Christoph is going to test this on CTGR and let us know how it goes.

https://github.com/jnordberg/browsernizr

**Comment by elseloop** [comment]
@stoff, did you ever wind up using [Browsernizr](https://github.com/jnordberg/browsernizr)? How'd it go?

I'll also add [Gulp Modernizr](https://github.com/doctyper/gulp-modernizr) to the list of options, which is essentially a Gulp wrapper for [Customizr](https://github.com/doctyper/customizr). (I'll also note that neither of those two repos have been updated in a couple years, though their download numbers on NPM are pretty solid.)

**Comment by stoff** [comment]
@elseloop YES! It was easy and simple to use and did exactly what I wanted.  it's been some time, however, I wonder if I should revisit it.  I have to say I really like the idea of Customizr, since it seems to do the same thing but automagically, although the crotchety old man in me feels like it is safer to have manual transmissions on really big trucks.  If you follow me.

**Comment by stoff** [comment]
yes
![YES](http://media.giphy.com/media/W9WSk4tEU1aJW/giphy.gif)
